### PR TITLE
Fix ui_vp_evidence_display being incorrectly displayed relative to the entire window instead of the viewport

### LINKED ISSUE #151: Evidence display is incorrectly drawn at 0,0 regardless of viewport location
**Steps to reproduce:**
  - Load a theme that uses a viewport located somewhere other than 0,0. Many 'large-screen' themes do this.
  - Make sure that `left_evidence_icon` and `right_evidence_icon` are appropriately located. (Extremely high numbers shouldn't be common here - remember, this is supposed to be relative to the viewport)
  - Present some evidence.

**Expected result:**
The evidence animation and subsequent display should play at the appropriate location relative to the viewport - at no point should it leave the viewport.

**Actual result:**
https://cdn.discordapp.com/attachments/615898062321090581/703863019377262692/unknown.png
As a bonus, should you position the evidence further to the right than the width of the viewport, Phil Swift will pay a visit to the Court Record to demonstrate the power of Flex Tape.
https://cdn.discordapp.com/attachments/353369729328611330/703857310015160390/unknown.png

### PR DESCRIPTION
**Pull request by in1tiate**
Additionally, adds a safeguard for a somewhat common issue related to broken default themes - evidence icons will now be displayed at 0,0 relative to the viewport and at 70px when the theme values `left_evidence_icon` and `right_evidence_icon` are not present.

Fixes #151.

### CODE REVIEW

File: src/aoevidencedisplay.cpp [hunk 1]
```diff
@@ -45,7 +48,10 @@ void AOEvidenceDisplay::show_evidence(QString p_evidence_image,
       ao_app->get_element_dimensions(icon_identifier, "courtroom_design.ini");
 
   evidence_icon->move(icon_dimensions.x, icon_dimensions.y);
-  evidence_icon->resize(icon_dimensions.width, icon_dimensions.height);
+  if (icon_dimensions.width == -1 || icon_dimensions.height == -1) // user's default theme is missing the values
+    evidence_icon->resize(70,70);
```
Comment (oldmud0) [inline src/aoevidencedisplay.cpp:52]:
```suggestion
    evidence_icon->resize(70, 70);
```

**Comment by oldmud0** [comment]
Patch is still valid for 2.8.

**Comment by stonedDiscord** [comment]
isn't this a duplicate of #151

**Comment by in1tiate** [comment]
> 
> 
> isn't this a duplicate of #151

It is not, as that is an issue and this is a pull request.

**Comment by Crystalwarrior** [comment]
Tested in kfo-ao2 branch, issue does not seem to be present with my theme.

**Comment by in1tiate** [comment]
I also can't reproduce the original issue on kfo-ao2.

**Pull request closed without merging**